pkg/cli/config.go

```diff
@@ -208,6 +211,9 @@ func (c *Config) RegisterCommandLineFlags() {
 		flag.StringVar(&c.Backend.FileDir, "keyring-file-dir", keyringDirectory, "keyring `directory` for file-backed keyring types")
 		flag.BoolVar(&c.Debug, "keyring-debug", false, "Enable keyring debug logging")
 	}
+	if c.Flags.isSet(FlagBLE) && c.Flags.isSet(FlagVIN) {
```

Comment:
I don't think we'd ever require FlagBLE without FlagVIN, but why is FlagVIN made explicitly required here?